bin/commands/ls.js

```diff
@@ -3,7 +3,7 @@
 const cacheArgs = require('../cacheArgs');
 const repoArgs = require('../repoArgs');
 
-const ls = require('../../src/commands/ls');
+const ls = require('../../src/commands/json-report');
```

Comment:
i kind of like the symmetry with them both being named "ls" - is it important to rename the src command file?